index.html

```diff
@@ -3752,6 +3752,10 @@ <h3>Parsing content</h3>
       <li>
         If |ndef|'s |typeNameField| is `1` ([=well-known type record=]):
         <ol>
+          <li>
+            Set |record|'s <a>isLocal</a> to `true` if |ndef| is a payload
```

Comment:
I suppose {`U`, `T`, `Sp`} types here should not be set `isLocal=true`.

Reply:
Not all sub-records must be local types.

Reply:
We need to decide what `isLocal` means: to denote that a record is in the payload of another record, of if the type is a local type.

Reply:
From a previous comment of @leonhsl it seems we want that `isLocal` means only a local type, not sub-recordness. However, when creating records, a local type may only exist in a containing record. But other than local types may be part of a message that is payload to a record.

Reply:
Yeah, that's what I read from the discussion you had in the issue, `isLocal` is to denote the record is a local type, to differentiate a "text" local type record (sub-record of course) from a TEXT sub-record.

I think we allow users to create local type records at all times, but we reject the `push()` promise when we found the top-level message contains a local type record.

Reply:
I was also thinking if we could consider an implicit context (no conflicts within a message), but  the NFC Forum RTD clearly says "NFC Forum Local Types are available for use within the context of another record". So local type may only occur in the payload of a record, but we should set `isLocal` only for those sub-records that actually have a local type. Done :).